docs/setup-cluster/deploy-cluster/basic.rst

```diff
@@ -0,0 +1,50 @@
+.. _basic-installation:
```

Comment:
Make the file name, the ref name, and the page title match please.

It's annoying to rename everywhere, yeah, but it's also annoying later if the names don't match up.  It's worth doing now.

Reply:
done.

Reply:
nope not done.

Reply:
i want to keep the filename "basic.rst", i'll update the anchor to match, but the title won't match for now (doing some analytics on it)

Reply:
marking as resolved